e2e-tests/insurance/cypress/e2e/journeys/your-buyer/connection-to-the-buyer/connection-to-the-buyer-page.spec.js

```diff
@@ -0,0 +1,136 @@
+import {
+  headingCaption, saveAndBackButton, field, yesRadio, noRadio, yesNoRadioHint, yesRadioInput,
+} from '../../../../../../pages/shared';
+import { BUTTONS, PAGES } from '../../../../../../content-strings';
+import { ROUTES, FIELD_VALUES } from '../../../../../../constants';
+import { YOUR_BUYER as FIELD_IDS } from '../../../../../../constants/field-ids/insurance/your-buyer';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+import { YOUR_BUYER_FIELDS } from '../../../../../../content-strings/fields/insurance/your-buyer';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER.CONNECTION_TO_THE_BUYER;
+
+const {
+  WORKING_WITH_BUYER: {
+    CONNECTED_WITH_BUYER,
+    CONNECTION_WITH_BUYER_DESCRIPTION,
+  },
+} = FIELD_IDS;
+
+const {
+  YOUR_BUYER: { COMPANY_OR_ORGANISATION, CONNECTION_TO_THE_BUYER, WORKING_WITH_BUYER },
+} = ROUTES.INSURANCE;
+
+const FIELDS = YOUR_BUYER_FIELDS.WORKING_WITH_BUYER;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Your Buyer - Connection with the buyer - As an exporter, I want to provide the details on trading history with the buyer of my export trade, So that UKEF can gain clarity on whether I have  trading history with the buyer as part of due diligence', () => {
```

Comment:
The description mentiones trading history with the buyer?

Reply:
thats come from the ticket